app/main.py

```diff
@@ -1,3 +1,56 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __add__(self, other: Union["Distance", int, float]) -> "Distance":
```

Comment:
Why do you need Union here? You use `|` in other functions, you can use same everywhere